lib/OpenQA/Schema/Result/Comments.pm

```diff
@@ -118,7 +118,10 @@ e.g. 'tag:0123:important:GM' returns a list of '0123', 'important' and 'GM'.
 =cut
 sub tag {
     my ($self) = @_;
-    $self->text =~ /\btag:(?<build>([-.@\d\w]+-)?[@\d\w]+):(?<type>[-@\d\w]+)(:(?<description>[@\d\w]+))?\b/;
+    $self->text =~ /\btag:(?<build>[-.@\d\w]+):(?<type>[-@\d\w]+)(:(?<description>[@\d\w]+))?\b/;
+    # note: calling this 'build' is a slight lie, because it may be
```

Comment:
I would not call it a lie as a "build" is an implicit specialization. If I say "build 1234" then obviously I mean the build as part of "version-product", etc. so it's still a "build".